client/src/components/App/index.js

```diff
@@ -1,12 +1,97 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Redirect,
+} from 'react-router-dom';
+import axios from 'axios';
+
 import './index.css';
+import AppBar from '../common/AppBar';
+import Limitation from '../common/limitation';
 
-function App() {
-  return (
-    <div>
-      <h1>Welcome in Applicants System</h1>
-    </div>
-  );
-}
+export default class App extends React.Component {
+  state = {
+    auth: null,
+  };
+
+  componentDidMount() {
+    axios
+      .get('/api/v1/auth')
+      .then((result) => {
+        this.setState({ ...result.data });
+      })
+      .catch(() => {
+        this.setState({ auth: false });
+      });
+  }
```

Comment:
search more in this point , as you know we don't have end point for that, so just think about that , how can we get 
`isAuthorized, isAdmin` & also how we can use it in checking ,cause writing the class by this way is not correct .